src/modules/ecore_evas/engines/wayland/ecore_evas_wayland_egl.c

```diff
@@ -5,23 +5,7 @@
 # include <string.h>
 # include <unistd.h>
 
-#ifdef _WIN32
```

Comment:
...wtf those definitions on arbitrary places

Reply:
They were done at those places at master :/

Reply:
Yeah, that's why "wtf"